src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/CourtCaseEventsListenerIntTest.kt

```diff
@@ -182,47 +167,6 @@ class CourtCaseEventsListenerIntTest : IntegrationTestBase() {
     }
   }
 
-  @Test
-  fun `should not push messages from Libra onto dead letter queue when processing fails because of could not serialize access due to read write dependencies among transactions`() {
-    // given
-    val publishRequest = PublishRequest.builder()
-      .topicArn(courtCaseEventsTopic?.arn)
-      .message(libraHearing())
-      .messageAttributes(
-        mapOf(
-          "messageType" to MessageAttributeValue.builder().dataType("String")
-            .stringValue(MessageType.LIBRA_COURT_CASE.name).build(),
-        ),
-      )
-      .build()
-
-    // when
-    val blitzer = Blitzer(100, 50)
-    try {
-      blitzer.blitz {
-        courtCaseEventsTopic?.snsClient?.publish(publishRequest)?.get()
-      }
-    } finally {
-      blitzer.shutdown()
-    }
-
-    // then
-    await untilCallTo {
-      cprCourtCaseEventsQueue?.sqsClient?.countMessagesOnQueue(cprCourtCaseEventsQueue!!.queueUrl)?.get()
-    } matches { it == 0 }
-
-    await untilCallTo {
-      cprCourtCaseEventsQueue?.sqsDlqClient?.countMessagesOnQueue(cprCourtCaseEventsQueue!!.dlqUrl!!)?.get()
-    } matches { it == 0 }
-
-    verify(telemetryService, times(1)).trackEvent(
-      eq(TelemetryEventType.NEW_CASE_PERSON_CREATED),
-      check {
-        assertThat(it["PNC"]).isEqualTo("2003/0011985X")
-      },
-    )
-  }
-
```

Comment:
I don't think we need both of these tests - they both use the `process` method which has the try/catch statement to handle this